### PR TITLE
Gabriel Rosengren

### PR DESCRIPTION
**Pull request by Gabriel-Rosengren**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,53 +3,225 @@ const state = {
...
+    } 
+    state.cart.push(item)
+    render()
+  })
+
+  div.append(img)
+  li.append(div, button)
+  ul.append(li)
+}
+
```
Comment (sarajbeazley) [inline index.js:95]: i really like the way you have structured the code here in the first 2 functions  - to select the list and cart and create elements to go inside those. it all reads v clearly

File: index.js [hunk 2]
```diff
@@ -3,53 +3,225 @@ const state = {
...
+    createFilterButton()
+  }
+  if(!document.querySelector('.sort-button')) {
+    createSortButton()
+  }
+  state.items.forEach(item => {
+
+    if (state.filteredByFruit) {
+      if (item.type != 'fruit') {
+        return
```
Comment (NoroAxper) [inline index.js:174]: smart statements, mi gusta (i likey)

File: index.js [hunk 3]
```diff
@@ -3,53 +3,225 @@ const state = {
...
+  render()
+}
+
+function sortAToZ() {
+  if(state.sortedAToZ === true) {
+    state.sortedAToZ = false
+    state.items.sort ((a, b) => {
+      if (a.id < b.id) return -1;
+      if (a.id > a.id) return 1;
+      return 0;
```
Comment (NoroAxper) [inline index.js:208]: nice, i spent like 30minutes trying to figure out how to do it with nested for loops, then just resorted to this haha

File: index.js [hunk 4]
```diff
@@ -3,53 +3,225 @@ const state = {
...
+    })
+  }
+  console.log('sorted A-Z = ', state.sortedAToZ)
+  render()
+}
+function init() {
+  state.items.forEach(item => item.quantity = 0)
+  render()
+}
+
```
Comment (NoroAxper) [inline index.js:226]: Great work, it was kinda easy to read along and understand, but looks like we both have the problem of just coding and not commenting, that is something for us both to work on

**Pull request closed without merging**